src/sas/qtgui/Plotting/Plotter.py

```diff
@@ -581,7 +581,7 @@ def onResetGraphRange(self):
         Resets the chart X and Y ranges to their original values
         """
         # Clear graph and plot everything again
-        mpl.pyplot.cla()
+        # mpl.pyplot.cla()
```

Comment:
These commented out calls can be safely removed.

Reply:
```suggestion 
```